commands/certmanager/post.go

```diff
@@ -0,0 +1,123 @@
+
+	cmd.AddStringFlag(FlagCertPath, "", "", "Specify the certificate itself from a file (required either this or --certificate)")
+	cmd.AddStringFlag(FlagCertChainPath, "", "", "Specify the certificate chain from a file (required either this or --certificate-chain)")
+	cmd.AddStringFlag(FlagPrivateKeyPath, "", "", "Specify the private key from a file (required either this or --private-key)")
+
+	return cmd
+}
+
+func GetPropertyWithFallback(c *core.CommandConfig, property string, propertyPath string) (string, error) {
+	propertyValue, err := c.Command.Command.Flags().GetString(property)
+	if err != nil {
+		return "", err
+	}
+	if propertyValue == "" {
+		propertyValuePath, err := c.Command.Command.Flags().GetString(propertyPath)
+		if err != nil {
+			return "", err
+		}
+		propertyBytes, err := os.ReadFile(propertyValuePath)
+		if err != nil {
+			return "", err
+		}
+		return string(propertyBytes), nil
+	}
+	return propertyValue, nil
+}
+
+func CmdPost(c *core.CommandConfig) error {
+	c.Printer.Verbose("Adding Certificate...")
+	var name, certificate, certificateChain, privateKey string
+
+	name, err := c.Command.Command.Flags().GetString(FlagCertName)
+	if err != nil {
+		return err
+	}
+	certificate, err = GetPropertyWithFallback(c, FlagCert, FlagCertPath)
+	if err != nil {
+		return err
+	}
+	certificateChain, err = GetPropertyWithFallback(c, FlagCertChain, FlagCertChainPath)
+	if err != nil {
+		return err
+	}
+	privateKey, err = GetPropertyWithFallback(c, FlagPrivateKey, FlagPrivateKeyPath)
+	if err != nil {
+		return err
+	}
+
+	createProperties.SetName(name)
+	createProperties.SetCertificate(certificate)
+	createProperties.SetCertificateChain(certificateChain)
+	createProperties.SetPrivateKey(privateKey)
+
+	Dto := sdkgo.NewCertificatePostDtoWithDefaults()
+
+	Dto.SetProperties(createProperties)
+
+	cert, _, err := c.CertificateManagerServices.Certs().Post(*Dto)
+	if err != nil {
+		return err
+	}
+
+	return c.Printer.Print(getCertPrint(nil, c, &[]sdkgo.CertificateDto{cert}))
+}
+
+// couldn't think of a better solution to this
+func PreCmdPost(c *core.PreCommandConfig) error {
+	err := errors.New("")
+	if !viper.IsSet(core.GetFlagName(c.NS, FlagCert)) && !viper.IsSet(core.GetFlagName(c.NS, FlagCertPath)) {
+		err = fmt.Errorf("%veither --%s or --%s must be set\n", err, FlagCert, FlagCertPath)
+	}
+	if !viper.IsSet(core.GetFlagName(c.NS, FlagCertChain)) && !viper.IsSet(core.GetFlagName(c.NS, FlagCertChainPath)) {
+		err = fmt.Errorf("%veither --%s or --%s must be set\n", err, FlagCertChain, FlagCertChainPath)
+	}
+	if !viper.IsSet(core.GetFlagName(c.NS, FlagPrivateKey)) && !viper.IsSet(core.GetFlagName(c.NS, FlagPrivateKeyPath)) {
+		err = fmt.Errorf("%veither --%s or --%s must be set\n", err, FlagPrivateKey, FlagPrivateKeyPath)
+	}
+	if !viper.IsSet(core.GetFlagName(c.NS, FlagCertName)) {
+		err = fmt.Errorf("%veither --%s must be set\n", err, FlagCertName)
```

Comment:
`FlagCertName` can just use cobra's markFlagAsRequired func, since it doesn't have a Path fallback